plone/app/layout/viewlets/common.py

```diff
@@ -229,7 +229,7 @@ class GlobalSectionsViewlet(ViewletBase):
 
     _opener_markup_template = (
         u'<input id="navitem-{uid}" type="checkbox" class="opener" />'
-        u'<label for="navitem-{uid}" role="button" aria-label="{title}"></label>'  # noqa: E 501
+        u'<label for="navitem-{uid}" aria-label="{title}"></label>'  # noqa: E 501
```

Comment:
The trailing `# noqa: E 501` can be removed given the line is shorter than 80 chars now.